src/Models/PointsStruct.php

```diff
@@ -42,4 +42,9 @@ public function toArray(): array
 
         return $points;
     }
+
+    public function count(): int
+    {
+        return count($this->points);
+    }
```

Comment:
I could not remember what was the purpose of counting here.

Reply:
There were a few cases in my own code where I needed to check if the PointsStruct was empty. Strictly, I should have done all these smaller tweaks in a separate PR. I've been making a few other tweaks that I'll submit as a separate PR. This includes making this and a few other structs `Countable`.

Reply:
It is okay for me. Doesn't seem to do any harm for now. :)